src/Test/Perf/util/test_util.csx

```diff
@@ -200,6 +216,41 @@ ProcessResult ShellOut(
     };
 }
 
+ProcessResult ShellOutUsingShellExecute(
```

Comment:
Is this really necessary for robocopy?  If so, we should probably either 

1. Have `bool shellExecute = false` as an optional parameter on the original ShellOut method.
2. Write our own directory-copy method.